styles/globals.css

```diff
@@ -365,6 +365,72 @@ blockquote {
   color: #4b5563;
 }
 
+/* Footer social hover effects */
+
+#twitter:hover,
+#twitter:focus {
+  transition: all 300ms;
+  transform: scale(1.05);
+  filter: brightness(1.1);
+  cursor: pointer;
+  color: white !important;
+  background: rgba(40, 40, 40, 1);
+}
+
+#github:hover,
+#github:focus {
+  transition: all 300ms;
+  transform: scale(1.05);
+  filter: brightness(1.1);
+  cursor: pointer;
+  color: white !important;
+  background: #f17f06;
+}
+
+#discord:hover,
+#discord:focus {
+  transition: all 300ms;
+  transform: scale(1.05);
+  filter: brightness(1.1);
+  cursor: pointer;
+  color: white !important;
+  background: #4b83fb;
+  background: -moz-linear-gradient(top, #4b83fb, #734df8);
+  background: -ms-linear-gradient(top, #4b83fb, #734df8);
```

Comment:
can we stick to one convention for colors, either hex or rgba. Better yet, use var names for maintainability - https://www.w3schools.com/css/css3_variables.asp